### DISCUSSION TITLE
How to adjust AI spawn distance

### DISCUSSION THREAD

**Discussion by BluFox2003** [KillahPotatoes/KP-Liberation#899] [Q&A]
Where is the parameter/function in the files to change how far away the AI spawn from the player

**Comment by GuanoCrunch** [comment]
If you unpack the PBO file, look for a file called kp_liberation_config.sqf
In that file you can adjust a lot of stuff and there is a variable for AI Sector activation.  Once you make what ever adjustments, you can repackage the PBO then use your new pbo to play.
